src/preferences/dialog/dlgpreflibrarydlg.ui

```diff
@@ -132,16 +132,19 @@
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
-       <widget class="QCheckBox" name="checkBox_SyncTrackMetadataExport">
+       <widget class="QCheckBox" name="checkBox_SyncTrackMetadata">
         <property name="text">
-         <string>Automatically write modified track metadata from the library into file tags</string>
+         <string>Synchronize track metadata with file tags</string>
+        </property>
+        <property name="toolTip">
+         <string>Automatically write modified track metadata into file tags and re-import metadata from updated file tags</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_SeratoMetadataExport">
         <property name="text">
-         <string>Write Serato Metadata to files (experimental)</string>
+         <string>Write Serato metadata to files (experimental)</string>
```

Comment:
Is this now also in both directions?

Reply:
Serato metadata is implicitly re-imported if available. Not sure what happens when new cue points/loops and color have been edited in Mixxx and the file is re-imported.

I have added a TODO.

Reply:
@Holzhaus Can you test/explain the situation, that we can adjust the text in one or the other direction?

Reply:
Done
![serato-metadata](https://user-images.githubusercontent.com/1474154/133607705-019ea414-1725-4436-a82c-1cd0194170bd.png)